### PR TITLE
docs: update custom domains, email templates, going to prod checlist

### PR DESCRIPTION
**Pull request by kangmingtay**

### CODE REVIEW

File: apps/docs/pages/guides/auth/auth-email-templates.mdx [hunk 1]
```diff
@@ -5,7 +5,7 @@ export const meta = {
   description: 'Learn how to configure the email templates on Supabase.',
 }
 
-You can customize the email messages used for the authentication flows. You can edit the following email templates:
+You can customise the email messages used for the authentication flows. You can edit the following email templates:
```
[the remark this answers is not in the document]
Comment (gregnr) [reply apps/docs/pages/guides/auth/auth-email-templates.mdx:8]: @kangmingtay maybe worth keeping the `customize` spelling for consistency across the docs?

File: apps/docs/pages/guides/auth/auth-email-templates.mdx [hunk 2]
```diff
@@ -45,6 +45,28 @@ To guard against this:
   The user should be brought to a page on your site where they can confirm the action by clicking a button.
   The button should contain the actual confirmation link which can be obtained from parsing the `confirmation_url={{ .ConfirmationURL }}` query parameter in the URL.
 
+### Redirecting the user to a server-side endpoint
+
+If you intend to use [Server-side rendering](/docs/guides/auth/server-side-rendering), you might want the email link to redirect the user to a server-side endpoint to check if they are authenticated before returning the page. However, the default email link will redirect the user after verification to the redirect URL with the session in the query fragments. Since the session is returned in the query fragments by default, you won't be able to access it on the server-side.
+
+You can customise the email link in the email template to redirect the user to a server-side endpoint successfully. For example:
```
[the remark this answers is not in the document]
Comment (gregnr) [reply apps/docs/pages/guides/auth/auth-email-templates.mdx:52]: Here as well.

**Review by gregnr — APPROVED** [review]

File: apps/docs/pages/guides/auth/auth-email-templates.mdx [hunk 3]
```diff
@@ -45,6 +45,28 @@ To guard against this:
   The user should be brought to a page on your site where they can confirm the action by clicking a button.
   The button should contain the actual confirmation link which can be obtained from parsing the `confirmation_url={{ .ConfirmationURL }}` query parameter in the URL.
 
+### Redirecting the user to a server-side endpoint
```
[the remark this answers is not in the document]
Comment (silentworks) [reply apps/docs/pages/guides/auth/auth-email-templates.mdx:48]: This looks good to me.

**Comment by kangmingtay** [comment]
update: will merge when the deployment for supabase auth that contains these changes go out

**Pull request merged**